internal/monitors/elasticsearch/client/indexstats.go

```diff
@@ -0,0 +1,176 @@
+package client
+
+/*
+ This nested struct contains all the index level stats of the following types
+ and will be used to retrieve node level metrics
+*/
+type IndexStatsOutput struct {
+	Shards        Shards                `json:"_shards"`
+	AllIndexStats IndexStats            `json:"_all"`
+	Indices       map[string]IndexStats `json:"indices"`
+}
+
+type Shards struct {
+	Total      *int64 `json:"total"`
+	Successful *int64 `json:"successful"`
+	Failed     *int64 `json:"failed"`
+}
+
+type IndexStats struct {
+	Primaries IndexStatsGroups `json:"primaries"`
+	Total     IndexStatsGroups `json:"total"`
+}
+
+type IndexStatsGroups struct {
```

Comment:
If you don't want to have to put a comment on all of these, just make them unexported (lowercase first letter).